frontend/src/pages/Orders.tsx

```diff
@@ -0,0 +1,10 @@
+import {Typography} from "antd";
+
+function Orders () {
+    return (
+        <div>
```

Comment:
Same here. you might not need the div's (except this Typo-thingy makes it necessary)